CQA DOCPLAN-175 flatten toc (#110808)
openshift/openshift-docs: modules/virt-connecting-vnc-console-web.adoc

// Module included in the following assemblies:
//
// * virt/managing_vms/virt-accessing-vm-consoles.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-connecting-vnc-console-web_{context}"]
= Connect to the VNC console by using the web console

[role="_abstract"]
You can connect to the VNC console of a virtual machine (VM) by using the {product-title} web console.

[NOTE]
====
If you connect to a Windows VM with a vGPU assigned as a mediated device, you can switch between the default display and the vGPU display.
====

.Procedure

. On the *Virtualization* -> *VirtualMachines* page, click a VM to open the *VirtualMachine details* page.
. In the navigation panel, right-click the virtual machine and select *Open Console*.
. Click the *Console* tab. The VNC console session starts automatically.
+
[IMPORTANT]
====

> Only one connection to the VNC console is possible at a time. If you try to create a second connection to the same VNC console, the console displays a warning and prompts you to disconnect the existing session before you create the new session.

Reviewer: ```suggestion
Only one connection to the VNC console is possible at a time. If you try to create a second connection to the same VNC console, a warning is displayed. You must disconnect the existing session before you create the new session.
```

> image::vnc-console-warning.png[VNC console connection warning]

Reviewer: Not sure if we need this screenshot here, especially if it is outdated

Author: removed
